### PR TITLE
Features/books

### PR DESCRIPTION
**Pull request by hansderly**
## Changes implemented

- Created Book class with methods.
- Built Label class with methods.
- Developed list book, list label and create book methods.

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    it 'should return true if cover state equal to bad or can_be_archived? is true' do
+      expect(book.can_be_archived?).to be(true)
+    end
+
+    it 'should return false if cover state equal to good or can_be_archived? is true' do
+      expect(book2.can_be_archived?).to be(false)
+    end
+  end
+end
```
Comment (jmonto55) [inline spec/book_spec.rb:24]: Excellent implementation of your tests for the Book class 💯. However, please kindly make sure that you add a test for the `add_item` method of your Label class too as requested in the requirement below 👇🏽.

- All methods visible in the diagram should be implemented:

  - add_item method (in all classes that have that method).
- Add unit tests for all implemented methods.

File: lib/modules/add_book.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    puts 'Enter the title'
+    title = gets.chomp
+
+    puts 'Enter the color'
+    color = gets.chomp
+
+    puts 'Enter the publish date (yyyy-mm-dd) : '
+    date = gets.chomp
+    # publish_date = Date.parse(date)
+    publish_date = date
```
Comment (jmonto55) [inline lib/modules/add_book.rb:12]: Great job building your AddBook module ✔️. However, please kindly remove the commented code here in order to follow Ruby best practices 🤓

**Review by jmonto55 — CHANGES_REQUESTED** [review]
Hi @hansderly 👋🏽, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
AlmostThereGIF (2)



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
 

- [ ]  Very DRY code ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/book_spec.rb (hunk 1 above)
Comment (hansderly) [reply spec/book_spec.rb:24]: Thanks @jmonto55 , I am working on it😉.

**Pull request merged**